lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2325,6 +2357,16 @@ function EDITOR:AC_Check( notimer )
 			self:AC_SetVisible( false )
 			return
 		end
+		
+		if suggestions[1].str( suggestions[1] ) == "vec" or suggestions[1].str( suggestions[1] ) == "vec4" then
```

Comment:
I guess they were coping the code from 5 lines above this.

@NickMBR `suggestions[1].str(suggestions[1])` in Lua is equivalent to `suggestions[1]:str()`, the `:` explicitly makes the object the first argument.